back/engines/commercial/analysis/app/controllers/analysis/web_api/v1/analyses_controller.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module Analysis
+  module WebApi
+    module V1
+      class AnalysesController < ApplicationController
+        before_action :set_analysis, except: %i[index create]
+
+        def index
+          authorize ::Analysis::Analysis
+          @analyses = policy_scope(::Analysis::Analysis).order(created_at: :asc)
+          @analyses = paginate(@analyses)
+          @analyses = @analyses.includes(:custom_fields)
+
+          if params[:project_id]
+            @analyses = @analyses.where(project_id: params[:project_id])
+          elsif params[:phase_id]
+            @analyses = @analyses.where(phase_id: params[:phase_id])
+          end
+
+          render json: linked_json(
+            @analyses,
+            WebApi::V1::AnalysisSerializer,
+            params: jsonapi_serializer_params,
+            include: [:custom_fields]
```

Comment:
When returning custom fields, it is often also desired to include the custom field options. Perhaps not needed here, but just checking.

Reply:
In this case it'll only be textual questions, so no need indeed.